perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -545,6 +545,23 @@ sub bulky_can_view_collection {
     return $c->user->id == $p->user_id;
 }
 
+sub bulky_can_view_cancellation {
```

Comment:
This function is duplicated above, so needs combining, not overwriting.

Reply:
I have made some changes so it is no longer a duplicate - Dave & I came to the agreement that non-admin users do not need to view a cancellation report (though if Peterborough decide otherwise, it should be easy enough to change).